src/app/static/src/js/components/runWorkflow/runWorkflowRun.vue

```diff
@@ -133,24 +144,23 @@
                         this.defaultMessage = "An error occurred while retrieving previously run workflows";
                     })
             },
-            handleValidation: function () {
-                let valid = !!this.workflowName;
+            handleWorkflowName: function (event: Event) {
+                console.log("hnadling workflow name")
+                const workflowName = (event.target as HTMLInputElement).value;
+                let valid = !!workflowName;
                 this.workflowNameError = "";
                 if (this.workflows.find(workflow =>
-                    workflow.name.toLowerCase() === this.workflowName.toLowerCase())) {
+                    workflow.name.toLowerCase() === workflowName.toLowerCase())) {
                     valid = false;
                     this.workflowNameError = "Workflow name already exists, please rename your workflow."
                 }
                 this.$emit("valid", valid);
-                if(this.workflowName) {
-                    this.$emit("update", {name: this.workflowName})
-                }
+                this.$emit("update", {name: workflowName});
```

Comment:
We still want to emit the change even if it is not valid -we'll get odd behvaviour if the wizard's metadata isn't updated to match the input - it will just invalid and unsubmittable.